generator/regions.cpp

```diff
@@ -570,8 +569,18 @@ Node::Ptr RegionsBuilder::BuildCountryRegionTree(Region const & country,
           (currRegion.Contains(firstRegion) ||
            currRegion.CalculateOverlapPercentage(firstRegion) > kAvaliableOverlapPercentage))
       {
-        (*itFirstNode)->SetParent(*itCurr);
-        (*itCurr)->AddChild(*itFirstNode);
+        // In general, we assume that a region with a large rank has a large area.
```

Comment:
In general, we assume that a region with the larger rank has the larger area.

Reply:
++